src/main/java/app/deploy/MainConfig.java

```diff
@@ -1,7 +1,6 @@
-package deploy;
+package app.deploy;
```

Comment:
Why is this needed?

Reply:
In `Application.java` above, I've split out the `@SpringBootApplication` annotation because it wasn't finding the controller in its component scan.  

Another option is to pass a list of packages like this:
`@ComponentScan({"deploy", "controllers"})`. 
In this case we would need to include any new packages to the list when they are added.